recipes/go-activation/conda_build_config.yaml

```
@@ -0,0 +1,39 @@
+cross_target_platform:
+  - linux-64                            # [linux]
+  - linux-aarch64                       # [linux]
+  - linux-ppc64le                       # [linux]
+  - osx-64                              # [linux or (osx and x86_64)]
```

Following should work right?
```suggestion
  - osx-64                              # [linux or (osx and x86_64) or (osx and arm64)]
```